### PR TITLE
[R-COMPAT] [q-mr1] Lose COPY_HEADERS, cleanup

### PR DESCRIPTION
**Pull request by ix5**
This should not break your proprietaries, @jerpelea .

### CODE REVIEW

File: rmnetctl/src/Android.mk [hunk 1]
```diff
@@ -30,8 +31,7 @@ LOCAL_MODULE := librmnetctl
 LOCAL_MODULE_TAGS := optional
 LOCAL_PRELINK_MODULE := false
 
-LOCAL_MODULE_PATH_32 := $(TARGET_OUT_VENDOR)/lib
-LOCAL_MODULE_PATH_64 := $(TARGET_OUT_VENDOR)/lib64
+LOCAL_VENDOR_MODULE: true
```
Comment (MarijnS95) [inline rmnetctl/src/Android.mk]:
```
vendor/qcom/opensource/dataservices/rmnetctl/src/Android.mk:34: error: writing to readonly directory: "LOCAL_VENDOR_MODULE"
```
You accidentally turned this one into a build rule :wink:

Comment (ix5) [reply rmnetctl/src/Android.mk]: ^^

Comment (ix5) [reply rmnetctl/src/Android.mk]: Also, whaddaya think? https://github.com/ix5/vendor-qcom-opensource-dataservices/tree/q-mr1-nocopyheaders-androidbp

**Review by MarijnS95 — APPROVED** [review]
Looks sane and compiles fine, can be merged IMO!

File: rmnetctl/src/Android.mk (hunk 1 above)
Comment (MarijnS95) [reply rmnetctl/src/Android.mk]: You know I like `.bp` :smile: 

Let me apply some review inline as to not diverge it here.

**Comment by ix5** [comment]
@jerpelea `:submarine-sonar-ping:`

**Pull request merged**